libs/api/domains/cms/src/lib/cms.elasticsearch.service.ts

```diff
@@ -175,6 +177,27 @@ export class CmsElasticsearchService {
     return response ? JSON.parse(response) : null
   }
 
+  async getSingleOrganizationSubpage(
+    index: string,
+    { slug, organizationSlug }: GetOrganizationSubpageInput,
+  ): Promise<OrganizationSubpage | null> {
+    // return an organization page by organization slug and subpage slug
+    const query = {
+      types: ['webOrganizationSubpage'],
+      tags: [
+        { type: 'slug', key: slug },
+        { type: 'organization', key: organizationSlug },
+      ],
+    }
+    const subpageResponse = await this.elasticService.getDocumentsByMetaData(
```

Comment:
I'd like to make a separate PR with this :) it needs to be changed in more places AFAIK, and I'd like to change `getSingleDocumentByMetaData` so that it returns a single document (currently returns the exact same thing as `getDocuments`)